plugin-docker/che-plugin-docker-client/src/test/java/org/eclipse/che/plugin/docker/client/DockerConnectorConfigurationTest.java

```diff
@@ -131,5 +143,98 @@ public void testDockerUriOnNonLinuxMissingProperties() throws Exception {
 
     }
 
+    /**
+     * Check that without property docker host ip from container is DEFAULT_LINUX_DOCKER_HOST_IP
```

Comment:
check if docker host ip from container is DEFAULT_LINUX_DOCKER_HOST_IP when property is not set